### PR TITLE
chore(register-component): just a bunch of cleanup of code and tests

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

File: plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.test.tsx [hunk 1]
```diff
@@ -40,35 +40,31 @@ const catalogApi: jest.Mocked<typeof catalogApiRef.T> = {
...
-          />
-        </ThemeProvider>
-      </ApiProvider>
-    </MemoryRouter>,
-  ),
-});
+const Wrapper = ({ children }: { children?: React.ReactNode }) => (
+  <MemoryRouter>
+    <ApiProvider
+      apis={ApiRegistry.from([
```
Comment (Rugvip) [inline plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.test.tsx:46]: `ApiRegistry.with()`! :D

**Review by Rugvip — APPROVED** [review]
👌

File: plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.test.tsx (hunk 1 above)
Comment (freben) [reply plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.test.tsx]: okeydokey!

**Review by Rugvip — APPROVED** [review]

**Comment by freben** [comment]
@stefanalund there is a balancing act to be had here regarding naming of things and texts/labels in the interface - it talks about registering components, but we now have a growing set of possible kinds that are moving front and center, such as APIs. However the catalog page focuses on showing only components still, so it isn't feature complete enough to purely refer to things as the more generic "entities". Let's start sketching on the next gen catalog pages.

**Pull request merged**